db/seeds/content_pages/communication_and_language.yml

```diff
@@ -34,6 +34,8 @@ interactions:
   parent: communication-and-language
   position: 1
   is_published: true
+  intro: |
```

Comment:
Just noting that the seed data here is only used when creating a new environment - this won't add the intro in the production environment unless we add it to the content page form.

Reply:
Understood, I thought it wise when bootstrapping. As a new comer I had presumed the ToC would be pre-populated . Would it be wise to add that to the seeds as well?

Reply:
The ToC came later (as an afterthought really) - I don't know what pages have it, but I think the presumption was most pages would not.  I'd say leave it.